src/puppetlabs/puppetdb/mq_listener.clj

```diff
@@ -76,16 +79,24 @@
 (defn create-metrics [prefix]
   (let [to-metric-name-fn #(metrics/keyword->metric-name prefix %)]
     {:processing-time (timer mq-metrics-registry (to-metric-name-fn :processing-time))
-     :retry-persistence-time (timer mq-metrics-registry (to-metric-name-fn :retry-persistence-time))
-     :generate-retry-message-time (timer mq-metrics-registry (to-metric-name-fn :generate-retry-message-time))
+     :message-persistence-time (timer mq-metrics-registry
+                                      (to-metric-name-fn :message-persistence-time))
      :retry-counts (histogram mq-metrics-registry (to-metric-name-fn :retry-counts))
+     :depth (counter mq-metrics-registry (to-metric-name-fn :depth))
+     :invalidated (counter mq-metrics-registry (to-metric-name-fn :invalidated))
      :seen (meter mq-metrics-registry (to-metric-name-fn :seen))
      :processed (meter mq-metrics-registry (to-metric-name-fn :processed))
      :fatal (meter mq-metrics-registry (to-metric-name-fn :fatal))
      :retried (meter mq-metrics-registry (to-metric-name-fn :retried))
      :discarded (meter mq-metrics-registry (to-metric-name-fn :discarded))}))
 
-(def metrics (atom {:global (create-metrics [:global])}))
+(def metrics (atom {}))
+
+(defn clear-counters!
```

Comment:
yeah, maybe. We might end up wanting it for HUP stuff depending on what we do there, but happy to move it.